modules/querying/pages/syntax-versions.adoc

```diff
@@ -1,9 +1,96 @@
 In TigerGraph 3.5, the default syntax version was changed to V2.
-This page provides a summary of the differences between V1 and V2, as well as guidance on how to change your V1 query into a V2 query.
+Currently, V2 remains the default syntax version. In a future release, V3 will become the default syntax version.
+This page provides a summary of the differences between V2 and V3, as well as V1 and V2. It also includes guidance on how to change your V1 query into a V2 query.
+
+== Differences between V2 and V3
+
+The table below highlights the differences between V2 and V3, focusing on the `FROM` clause.
+Note that V3 still supports V2 style syntax, for backward compatibility.
+
+|===
+|Feature|V2 Style |V3 Style
+
+|Vertex-Edge-Vertex Markers
+|Use `source_vset -(edge_set)- target_vset` notation. Vertices and edges are separated without additional symbols for vertices (e.g., `Person -(FRIENDS)- Colleague`).
+|Use `(source_vset) -[edge_set]- (target_vset)` notation, improving clarity and aligning with GQL style. Vertices are enclosed in parentheses () and edges in square brackets [] (e.g., `(Person) -[FRIENDS]- (Colleague)`), making the structure visually clearer.
+
+|Alias Order
+|Object sets and aliases follow `source_vset:S -(edge_set:E)- target_set:T`.
+|Object sets and aliases follow `(S:source_vset) -[E:edge_set]- (T:target_set)`, reversing the order of alias vs. object set.
+
+|Edge Direction Notation
+|Edge direction is indicated using arrowheads inside the parentheses for directed edges, e.g., `-(FRIENDS>:f)- or -(:e)-`.
+|Edge direction uses square brackets with arrowheads, e.g., `[FRIENDS>]` for directed edges. This aligns with modern graph query standards, providing consistency.
+
+|Filtering Within Edge Descriptions
+|Filters are applied outside the `FROM` clause, e.g.,
+`SELECT Person FROM Person -(FRIENDS)- Colleague WHERE FRIENDS.length > 2`.
+|Filters are integrated directly into the edge pattern within the FROM clause, e.g.,
+`SELECT Person FROM (Person) -[FRIENDS WHERE FRIENDS.length > 2]-> (Colleague)`.
+
+|Repeating Patterns
+|Supported using quantifiers like `*m..n` for repeating hops, e.g., `-[:FRIENDS*1..3]->`.
+|Supports GQL-style quantifiers {m,n} for variable-length patterns, e.g., `-[FRIENDS]->{1,3}`. Both options are available in V3 for flexibility.
+
+|Multi-Hop Patterns
+|Multi-hop patterns are expressed through a sequence of single hops. Each hop is written explicitly in the query.
+|Multi-hop patterns can be expressed concisely using quantifiers, improving query simplicity and readability, e.g., `(Person) -[FRIENDS*]-> (Celebrity)`.
+
+|Symbols for Edge Direction
+|Supports `<`, `>`, and no arrowhead for undirected edges, e.g., `-(Edge:e)-`.
+|Supports `<`, `>`, and `~` for undirected edges in square brackets, e.g., `[Edge~]`.
+
+|Edge Type Wildcards
+|Use `_` or `ANY` for any edge type, directed or undirected, e.g., `-(ANY:e)-`.
+|Similar to V2 but uses square brackets, e.g., `[ANY~]`.
+
```

Comment:
This row should be merged with the "Edge Direction Notation" row above, but this title is better "Symbols for Edge Direction".